c31-pryhaven/nutshell2
# added a delete function for each event. It is working as intended.

# Description

Added a delete button for each event.  The user can now remove events that have passed.

Applies to issue # (issue number)
#3 

## Type of change

Confirm that the delete button works by deleting any event.

Please delete options that are not relevant.


# Testing Instructions

run git fetch --all in terminal
pull down `my-event-del` from github.
take data from the example database and plug it into nutshell.json.
run the command `json-server -p 8088 -w nutshell.json at ./nutshell2/api`
go to root path and run npm start
go to browser and click on events on the nav bar.
click on the delete button for any event.  when button is clicked it should remove the event.



# Checklist:
